src/openrct2/paint/tile_element/Paint.Wall.cpp

```diff
@@ -89,46 +81,41 @@ static void PaintWallDoor(
         case 0:
         {
             BoundBoxXYZ bbR1 = { { 1, 1, height + 1 }, { 1, 3, bbHeight - 5 } };
-            BoundBoxXYZ bbR2 = { { 1, 1, height + bbHeight - 9 }, { 1, 28, 3 } };
-
-            BoundBoxXYZ bbL = { { 1, 1, height + 1 }, { 1, 28, bbHeight } };
+            BoundBoxXYZ bbR2 = { { 1, 1, height + bbHeight - 4 }, { 1, 28, 3 } };
```

Comment:
these offsets were wrong when we implemented the function. I checked it against the assembly implementor missed that there was a pop eax.